R/def_minerals.R

```diff
@@ -9,35 +9,39 @@
 #' @param mineral_type \code{character}. The type of mineral (e.g.,
 #'   "feldspar") to return a definition for.
 #' @param element \code{character}. The chemical element abbreviation of a
-#'   mineral to return a definition for (e.g., "CaCO3"). Note: abbreviations
+#'   mineral (e.g., "CaCO3") to return a definition for. Note: abbreviations
 #'   are case sensitive.
 #'
 #' @return A \code{data.frame} containing the following columns:
 #' \itemize{
-#'   \item \code{mineral_id}: The unique identification number of the mineral.
+#'   \item \code{mineral_id}: The unique identification number(s) of the
```

Comment:
Does this need the (s) here and below (L17 & L18)? I guess you will only ever have one number in each row?

Reply:
True! Agreed, these can be left out.